tests/atomizer.js

```diff
@@ -1071,6 +1071,34 @@ describe('Atomizer()', function () {
             const result = atomizer.getCss(config);
             expect(result).to.equal(expected);
         });
+        it ('has basic suppport for css variables', function () {
+            const atomizer = new Atomizer();
+            const config = {
+                breakPoints: {
+                    sm: '@media screen and (min-width:700px)'
+                },
+                classNames: [
+                    'C(--brand-color)',
+                    'Fz(--font-size)',
```

Comment:
Could also add a negative test case, eg `C(--)` should not be valid

Reply:
added